tests/postgres_tests/test_indexes.py

```diff
@@ -1,8 +1,31 @@
-from django.contrib.postgres.indexes import GinIndex
+from django.contrib.postgres.indexes import BrinIndex, GinIndex
 from django.db import connection
+from django.test import skipUnlessDBFeature
 
 from . import PostgreSQLTestCase
-from .models import IntegerArrayModel
+from .models import CharFieldModel, IntegerArrayModel
+
+
+@skipUnlessDBFeature('has_brin_index_support')
+class BrinIndexTests(PostgreSQLTestCase):
+
+    def test_repr(self):
+        index = BrinIndex(fields=['title'], pages_per_range=4)
+        another_index = BrinIndex(fields=['title'])
+        self.assertEqual(repr(index), "<BrinIndex: fields='title', pages_per_range=4>")
+        self.assertEqual(repr(another_index), "<BrinIndex: fields='title'>")
+
+    def test_not_eq(self):
+        index = BrinIndex(fields=['title'])
+        index_with_page_range = BrinIndex(fields=['title'], pages_per_range=16)
+        self.assertNotEqual(index, index_with_page_range)
+
+    def test_deconstruction(self):
+        index = BrinIndex(fields=['title'], name='test_title_brin')
+        path, args, kwargs = index.deconstruct()
+        self.assertEqual(path, 'django.contrib.postgres.indexes.BrinIndex')
+        self.assertEqual(args, ())
+        self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_brin', 'pages_per_range': None})
```

Comment:
Added in a new test, along with a test to test for an invalid `pages_per_range` parameter. Probably why Github doesn't think any change took place.